pytorch_lightning/strategies/deepspeed.py

```diff
@@ -767,13 +767,13 @@ def load_checkpoint(self, checkpoint_path: _PATH) -> Dict[str, Any]:
         return client_state
 
     @property
-    def lightning_restore_optimizer_and_schedulers(self) -> bool:
+    def lightning_restore_optimizer(self) -> bool:
         # managed by DeepSpeed
         if self.load_full_weights and self.zero_stage_3 and self.lightning_module.trainer.state.fn == TrainerFn.FITTING:
             rank_zero_warn(
-                "A single checkpoint file has been given. This means optimizer states and "
-                "scheduler states can not be restored. If you'd like to restore these states, you must "
-                "provide a path to the originally saved DeepSpeed checkpoint."
+                "A single checkpoint file has been given. This means optimizer states can not be restored."
```

Comment:
nit (i know this was there before):

```suggestion
                "A single checkpoint file has been given. This means optimizer states cannot be restored."
```

> Both cannot and can not are perfectly fine, but cannot is far more common and is therefore recommended, especially in any kind of formal writing.

https://www.merriam-webster.com/words-at-play/cannot-vs-can-not-is-there-a-difference#:~:text=Both%20cannot%20and%20can%20not,general%2C%20it%20is%20somewhat%20informal.